### PR TITLE
Jk/corp prospect

### PR DESCRIPTION
**Pull request by jkerr321**
TODO: 
- finish styling on company logos as they're not working responsively right now
- rationalise some of the styling cos i think it's a bit messy

screencapture-local-ft-5050-products-1505223496445

### CODE REVIEW

File: scss/components/barriers/_corp-prospect-trial.scss [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+
+		@include oGridRespondTo(S) {
+				padding: 0 0 0 17px;
+		}
+
+	}
+
+	li::before {
+		@include oGridRespondTo(S) {
+			content: "•";
```
Comment (doramatadora) [inline scss/components/barriers/_corp-prospect-trial.scss]: Really picking nits, but could we use the Unicode? `content: "\2219"`

File: scss/components/barriers/_corp-prospect-trial.scss [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+		align-items: center;
+		padding: 0;
+		margin-bottom: 12px;
+
+		@include oGridRespondTo(M) {
+			padding: 0 24px 0 0;
+		}
+	}
+
+	img {
```
Comment (doramatadora) [inline scss/components/barriers/_corp-prospect-trial.scss]: This is absolutely fine but got me thinking if there is any preference for defining this in the view code, perhaps something like `data-o-grid-colspan="12 S7 M7"`?

File: scss/components/barriers/_corp-prospect-trial.scss [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+
+	span {
+		display: block;
+	}
+
+}
+
+.corp-trial__enquire-button {
+	text-align: center;
+	margin-top: 12px;
```
Comment (doramatadora) [inline scss/components/barriers/_corp-prospect-trial.scss]: shorthand here? `margin: 12px 0 36px;`

**Review by doramatadora — APPROVED** [review]
I would replace the bullet with unicode - otherwise 👍

File: scss/components/barriers/_corp-prospect-trial.scss (hunk 3 above)
Comment (jkerr321) [reply scss/components/barriers/_corp-prospect-trial.scss]: I tried that but there is some built in styling in the row or button or something that didn't respond to the short hand, but only to the specific `bottom` and `top` properties. Although now I write this that sounds a bit ridiculous!

**Pull request merged**